packages/wallet/src/pages/index.tsx

```diff
@@ -65,6 +66,7 @@ export default function Home({ accounts }: HomeProps) {
           ))}
         </div>
       </div>
+      <SmallBubbles className="absolute inset-x-0 bottom-0 block w-full lg:hidden" />
```

Comment:
This is going over the account cards.
<details>
<summary>Preview </summary>
<img width="539" alt="image" src="https://user-images.githubusercontent.com/60383558/221189355-c87267ea-ff9b-45b8-b03d-6485469e07cc.png">

</details>